packages/angular_devkit/build_webpack/test/app-shell/app-shell_spec_large.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { normalize, virtualFs } from '@angular-devkit/core';
+import { tap } from 'rxjs/operators';
+import { host, runTargetSpec } from '../utils';
+
+
+describe('AppShell Builder', () => {
+  beforeEach(done => host.initialize().subscribe(undefined, done.fail, done));
+  afterEach(done => host.restore().subscribe(undefined, done.fail, done));
+  //     BrowserModule.withServerTransition({ appId: 'some-app' })
```

Comment:
Nit: this comment looks like it was forgotten here.

Reply:
Done.